test/PanelSpec.js

```diff
@@ -139,77 +140,92 @@ describe('<Panel>', () => {
 
   it('Should be closed', () => {
     const instance = ReactTestUtils.renderIntoDocument(
-      <Panel collapsible expanded={false} header="Heading">
+      <Panel collapsible expanded={false} onToggle={arg => arg}>
+        <Panel.Heading>
+          <Panel.Title toggle>
+            Heading
+          </Panel.Title>
+        </Panel.Heading>
         Panel content
       </Panel>
     );
     const collapse = ReactDOM.findDOMNode(instance).querySelector('.panel-collapse');
     assert.notOk(collapse.className.match(/\bin\b/));
   });
 
-  it('Should call onSelect handler', (done) => {
-    function handleSelect(key) {
-      assert.equal(key, '1');
+  it('Should call onToggle handler', (done) => {
+    function handleToggle(active) {
+      assert.equal(active, true);
       done();
     }
     const instance = ReactTestUtils.renderIntoDocument(
-      <Panel collapsible onSelect={handleSelect} header="Click me" eventKey="1">
+      <Panel collapsible onToggle={handleToggle} eventKey="1">
+        <Panel.Heading>
+          <Panel.Title toggle>
+            Heading
+          </Panel.Title>
+        </Panel.Heading>
         Panel content
       </Panel>
     );
     const title = ReactTestUtils.findRenderedDOMComponentWithClass(instance, 'panel-title');
     ReactTestUtils.Simulate.click(title.firstChild);
   });
 
-  it('Should obey onSelect handler', () => {
-    function handleSelect(key, e) {
-      if (e.target.className.indexOf('ignoreme') > -1) {
-        e.selected = false;
-      }
-    }
+  it('Should obey onToggle handler', () => {
     const header = (
       <div>
         <span className="clickme">Click me</span>
         <span className="ignoreme">Ignore me</span>
       </div>
     );
     const instance = ReactTestUtils.renderIntoDocument(
-      <Panel collapsible onSelect={handleSelect} header={header} eventKey="1">
+      <Panel collapsible onToggle={arg => arg} eventKey="1">
+        <Panel.Heading>
+          <Panel.Title toggle>
+            {header}
+          </Panel.Title>
+        </Panel.Heading>
         Panel content
       </Panel>
     );
     const panel = ReactTestUtils.findRenderedComponentWithType(instance, Panel);
     ReactTestUtils.Simulate.click(
-      ReactTestUtils.findRenderedDOMComponentWithClass(instance, 'ignoreme')
+      ReactTestUtils.findRenderedDOMComponentWithClass(instance, 'clickme')
     );
-    assert.notOk(panel.state.expanded);
+    assert.ok(panel._values.expanded);
     ReactTestUtils.Simulate.click(
-      ReactTestUtils.findRenderedDOMComponentWithClass(instance, 'clickme')
+      ReactTestUtils.findRenderedDOMComponentWithClass(instance, 'ignoreme')
     );
-    assert.ok(panel.state.expanded);
+    assert.notOk(panel._values.expanded);
```

Comment:
instead of checking this, use a `spy` for onToggle and test that the argument `expanded` is correct as you clic